.github/workflows/e2e.yml

```
@@ -8,10 +8,116 @@ on:
+
+      - name: Prepare yarn 4
+        run: corepack prepare yarn@4.9.1 --activate
+
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          large-packages: false
+
+      - name: Compute app tree hash
+        id: app-tree-hash
+        run: echo "hash=$(git rev-parse HEAD:app)" >> $GITHUB_OUTPUT
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:42: Double quote to prevent globbing and word splitting

(shellcheck)